yarn.lock

```diff
@@ -2536,6 +2536,15 @@
     "@sentry/utils" "7.13.0"
     tslib "^1.9.3"
 
+"@sentry/core@7.16.0":
+  version "7.16.0"
+  resolved "https://registry.yarnpkg.com/@sentry/core/-/core-7.16.0.tgz#60f9b54ef2ec524176b329e1d15be39c36da5953"
```

Comment:
What are the changes to yank.lock for? Just curious

Reply:
Probably because there are new versions of packages.

Reply:
Yes!